recipes/genologics/build.sh

```diff
@@ -0,0 +1,2 @@
+$PYTHON setup.py install --single-version-externally-managed --root=/
```

Comment:
Sorry, just see this now. Can you please remove the `--root=/`. I don't think this is needed.